src/main/kotlin/build/buf/intellij/index/ModuleKeyIndex.kt

```diff
@@ -77,7 +77,7 @@ class ModuleKeyIndex : ScalarIndexExtension<ModuleKey>() {
         return try {
             val moduleFullName = bufModuleItem["name"].let { moduleFullName ->
                 if (moduleFullName != null) { // v2
-                    ModuleFullName.parse(moduleFullName).getOrThrow()
+                    ModuleFullName.parse(moduleFullName).getOrNull() ?: return null
```

Comment:
We need to be more flexible when dealing with malformed buf.lock files and just fail to index them if parsing fails for any reason.